app/models/lighthouse_document.rb

```diff
@@ -3,11 +3,13 @@
 require 'common/models/base'
 require 'pdf_info'
 
+# [wipn8923] replaces evss_claim_document.rb ?
 class LighthouseDocument < Common::Base
```

Comment:
Confirmed - this is replacing EVSSClaimDocument  1:1. Some of this behavior may have been blindly copy pasted and may not need to apply to Lighthouse but difficult to tell without deep digging.

Was able to work with this successfully in my branch when updating the `SubmitUploads` worker